### PR TITLE
tests: All GLSL shader use raw strings

### PR DESCRIPTION
**Pull request by sfricke-samsung**
Can now enforce this as the standard for all future GLSL tests

### CODE REVIEW

File: tests/vklayertests_pipeline_shader.cpp [hunk 1]
```diff
@@ -6725,14 +6750,15 @@ TEST_F(VkLayerTest, SubgroupSupportedProperties) {
...
-            "void main(){\n"
-            "   uvec4 a = subgroupPartitionNV(ssbo.x);\n"  // forces OpGroupNonUniformPartitionNV
-            "   gl_Position = vec4(float(a.x));\n"
-            "}\n";
+        vsSource = R"glsl(
+            #version 450
+            #extension GL_NV_shader_subgroup_partitioned: enable
+            layout(set = 0, binding = 0) buffer StorageBuffer { float x; uint y; } ssbo;
+            void main(){
+               uvec4 a = subgroupPartitionNV(ssbo.x);\n"  // forces OpGroupNonUniformPartitio
```
Comment (ncesario-lunarg) [inline tests/vklayertests_pipeline_shader.cpp:6758]: Looks like a `\n"` snuck in here.

**Review by ncesario-lunarg — APPROVED** [review]
Aaaah, I've been wanting to do this for a while. Thanks @sfricke-samsung! I think with the removal of the one "left behind `\n"`" this should be GTG!

File: tests/vklayertests_pipeline_shader.cpp (hunk 1 above)
Comment (sfricke-samsung) [reply tests/vklayertests_pipeline_shader.cpp:6758]: Thanks, I was using CI to catch these... let me fix this quick

File: tests/vklayertests_pipeline_shader.cpp [hunk 2]
```diff
@@ -6839,15 +6865,17 @@ TEST_F(VkLayerTest, SubgroupExtendedTypesEnabled) {
...
-        "#extension GL_KHR_shader_subgroup_arithmetic : enable\n"
-        "#extension GL_EXT_shader_subgroup_extended_types_float16 : enable\n"
-        "#extension GL_EXT_shader_explicit_arithmetic_types_float16 : enable\n"
-        "layout(local_size_x = 32) in;\n"
-        "void main() {\n"
-        "   subgroupAdd(float16_t(0.0));\n"
-        "}\n";
+    char const *csSource = R"glsl(
+        #version 450
+        #version 450
```
Comment (ncesario-lunarg) [inline tests/vklayertests_pipeline_shader.cpp:6870]: Duplicate version pragmas.

**Pull request merged**